extension-manifest-v3/src/utils/dnr-converter.js

```diff
@@ -18,12 +18,16 @@ export function createDocumentConverter() {
 
     window.addEventListener('message', (event) => {
       const requestId = event.data.rules.shift().condition.urlFilter;
+      let { rules, errors } = event.data;
 
-      if (__PLATFORM__ === 'safari') {
-        event.data.rules = event.data.rules.map(getCompatRule).filter(Boolean);
-      }
+      requests.get(requestId)({
```

Comment:
Here I struggled first to understand what the effect is (same with the original version). Took me a bit to realize that "requests" is a mapping to callbacks, which should be triggered when the results of a conversion are available.

Maybe it would help the reader to name it to something like "requestResolvers"?

Reply:
I'll look into it, but this part was at least touched by me in this PR ;)